### PR TITLE
Steam launch options

### PR DESCRIPTION
**Pull request by pchome**
tl;dr: Allow to use `protonfixes` from Steam launch options.

In particular:
1. Adds `default.py` "gamefix" support, both "global" and "local". Can be used to set some environment variables for all games. For example, I use this for `util.set_environment('__GL_SHOW_GRAPHICS_OSD', '1')`.
  Only one of them used at a time, if exist, "local" have higher priority.

2. Adds `util.get_game_exe_name()`.

3. Adds `util.winecfg()`, `util.regedit()` and `util.control()`, which are aliases for corresponding Wine tools.

4. Adds `-pf_` prefixed aliases (**p**roton**f**ixes), to use as command line parameters.
  Very basic, but something like `-pf_tricks=dsound  -pf_tricks=vcrun2017 -pf_winecfg` in Steam launch options should work. Commands after `-pf_tricks=` are normal `winetricks` commands.

Well, all this changes are not very useful for "gamefixes", rather for "developing" them or for messing with "localfixes". But, I guess, users may want to have them combined this way.

### CODE REVIEW

**Comment by pchome** [comment]
I wrote this code using `mc`, Travis. Please, don't judge me so hard :)

Ok, I'll check this later.

**Comment by simons-public** [comment]
Love it
